core/settings/qPrefPrivate.cpp

```diff
@@ -9,3 +9,10 @@ qPrefPrivate *qPrefPrivate::instance()
 	static qPrefPrivate *self = new qPrefPrivate;
 	return self;
 }
+
+void qPrefPrivate::copy_txt(const char **name, const QString& string)
```

Comment:
1) Am I missing the member variable that this is using or why is this non-static?
2) Idiomatic C++ would be
```
void qPrefPrivate::copy_txt(const char *&name, const QString& string)
```

Reply:
seeabove

Reply:
for the idiot here... what's the difference between `const char **name` and `const char *&name` in this context?

Reply:
In this context the difference is purely syntactical - i.e. irrelevant.

In other contexts there are subtle differences between pointers and references. Notably: you can't "reseat" references - they reference the same object throughout their lifetime. And there is no null-reference. From a conceptual point-of-view, a reference is just a name for a different object.